app/javascript/src/clinics.js

```diff
@@ -0,0 +1,72 @@
+const getClinics = () => $.get('/clinics', (data, status) => {
+  if (status === 'success') {
+    $.when(data);
+  }
+}, 'json');
+const filterClinicsByNAF = () => {
+  const checked = $('#naf_filter').prop('checked');
+  $('#patient_clinic_id > option').each((_index, element) => {
+    if ($(element).attr('data-naf') === 'false') {
+      $(element).attr('disabled', checked);
+    }
+  });
+};
+
+const mapNAFtoClinic = (clinics) => {
+  clinics.forEeach((clinic) => {
```

Comment:
minor spelling error here:

```suggestion
  clinics.forEach((clinic) => {
```